gradle/libs.versions.toml

```diff
@@ -24,7 +24,7 @@
 ###
 
 [versions]
-com_amazonaws_aws_kms = "1.12.538"
+com_amazonaws_aws_kms = "1.12.668" # Addresses https://github.com/TBD54566975/web5-kt/security/advisories/GHSA-g6qj-pj38-2465
```

Comment:
i don't think we need a comment here coz this PR will be merged into main and the PR title which becomes the commit message describes why this version was bumped

Reply:
Updated PR to accept @frankhinek suggestion, rebased